### DISCUSSION TITLE
gleam: Request to Update Syntax Highlighting for Gleam Language

### DISCUSSION THREAD

**Discussion by mfreeman451** [community/community#75283] [Other Feature Feedback, Questions, & Ideas] [answered]
Would it be possible for the GitHub team to update the syntax highlighting to the latest version of the tree-sitter grammar for Gleam? This update would greatly benefit the growing community of Gleam developers and enhance our coding and collaboration experience on the platform.

https://github.com/gleam-lang/tree-sitter-gleam

Example of it not working correctly: https://github.com/lpil/aws4_request/blob/main/src/aws4_request.gleam

Thanks

**Comment by rewinfrey** [comment, marked as answer]
👋 @mfreeman451, thanks for letting us know about the recent release version of `tree-sitter-gleam`, and for providing an example of a Gleam file exhibiting the problematic highlighting behavior. You should now see improved Gleam syntax highlighting (including the file you've provided as an example).

Thanks again!

**Reply by mfreeman451** [reply]
Thanks!
